### PR TITLE
fix: escaped quotes

### PR DESCRIPTION
**Pull request by Xavientois**

### CODE REVIEW

File: src/jobs/cdk/cdk-deploy.yml [hunk 1]
```diff
@@ -8,9 +8,11 @@ steps:
...
       name: Install CDK
-      command: "sudo npm install -g aws-cdk\n"
+      command: sudo npm install -g aws-cdk
   - run:
       environment:
         GOPRIVATE: github.com/voiceflow
       name: CDK Deploy
-      command: "git config --global url.\"https://$GITHUB_TOKEN:x-oauth-basic@github.com/\".insteadOf \"https://github.com/\"\ncdk deploy --require-approval never\n"
+      command: |
+        git config --global url.https://$GITHUB_TOKEN:x-oauth-basic@github.com/.insteadOf https://github.com/
```
Comment (xavidop) [inline src/jobs/cdk/cdk-deploy.yml:17]: missing quotes here

File: src/jobs/cdk/cdk-go-deploy.yml [hunk 2]
```diff
@@ -3,9 +3,11 @@ steps:
...
       name: Install CDK
-      command: "sudo npm install -g aws-cdk\n"
+      command: sudo npm install -g aws-cdk
   - run:
       environment:
         GOPRIVATE: github.com/voiceflow
       name: CDK Deploy
-      command: "git config --global url.\"https://$GITHUB_TOKEN:x-oauth-basic@github.com/\".insteadOf \"https://github.com/\"\ncdk deploy --require-approval never\n"
+      command: |
+        git config --global url.https://$GITHUB_TOKEN:x-oauth-basic@github.com/.insteadOf https://github.com/
```
Comment (xavidop) [inline src/jobs/cdk/cdk-go-deploy.yml:12]: missing quotes here

File: src/jobs/cdk/cdk-deploy.yml (hunk 1 above)
Comment (xavidop) [reply src/jobs/cdk/cdk-deploy.yml:17]: git config --global url."https://$GITHUB_TOKEN:x-oauth-basic@github.com/".insteadOf "https://github.com/"

File: src/jobs/cdk/cdk-go-deploy.yml (hunk 2 above)
Comment (xavidop) [reply src/jobs/cdk/cdk-go-deploy.yml:12]: git config --global url."https://$GITHUB_TOKEN:x-oauth-basic@github.com/".insteadOf "https://github.com/"

Comment (Xavientois) [reply src/jobs/cdk/cdk-go-deploy.yml:12]: I noticed it had the quotes before, but I believe they are unnecessary in this case: https://stackoverflow.com/a/11383587/4382974

File: src/jobs/cdk/cdk-deploy.yml (hunk 1 above)
Comment (Xavientois) [reply src/jobs/cdk/cdk-deploy.yml:17]: I removed them intentionally

File: src/jobs/cdk/cdk-go-deploy.yml (hunk 2 above)
Comment (Xavientois) [reply src/jobs/cdk/cdk-go-deploy.yml:12]: I am fine to put them back though

**Review by xavidop — APPROVED** [review]

**Pull request merged**